rfcs/0000-single-cluster-dnspolicy.md

```diff
@@ -0,0 +1,272 @@
+      recordTTL: 60
+      recordType: A
+      targets:
+        - 172.31.200.0
+```
+
+Note: `LoadBalanced` is the current default for DNSPolicy and more details about it can be found in the original [DNSPolicy rfc](https://github.com/Kuadrant/architecture/blob/main/rfcs/0003-dns-policy.md) 
+
+### DNSRecord.spec.providerRef
+
+More details of `providerRef` found in [DNSPolicy.spec.providerRef](#dnspolicyspecproviderref)
+
+The DNSRecord API is updated to remove the `managedZone` reference in favour of directly referencing the `providerRef` credentials instead. The DNSRecord reconciliation will be unchanged except for loading the provider client from `providerRef` credentials.
+
+The DNSPolicy reconciliation will be updated to remove the requirement for a ManagedZone resource to be created before a DNSPolicy can create dns records for it, instead it will be replaced in favour of just listing available zones directly in the currently configured dns provider. If no matching zone is found, no DNSRecord will be created.
+
+There is a potential for a DNSRecord to be created successfully, but then a provider updated to remove access. In this case it is the responsibility of the DNSPolicy controller to report appropriate status back to the policy and target resource about the failure to process the record. More details on how status will be reported can be found in [rfc-0004](https://github.com/Kuadrant/architecture/blob/main/rfcs//0004-rlp-status.md)
+
+### DNSRecord.spec.zoneID
+
+The `zoneID` field is mandatory and contains the provider specific id of the hosted zone that this record should be published into. 
+
+The DNSRecord reconciliation will use this zone when creating/updating or deleting endpoints for this record set. 
+
+The `zoneID` should not change after being selected during initial creation and as such will be marked as immutable.
+
+### ManagedZone.spec.providerRef
+
+More details of `providerRef` found in [DNSPolicy.spec.providerRef](#dnspolicyspecproviderref)
+
+Replaces the existing `dnsProviderSecretRef` for consistency with other resources that require a provider reference (DNSRecord and DNSPolicy). 
+
+In the case of a ManagedZone a providerRef kind of type `ManagedZone` will not be allowed and will be rejected during create/update.  
+
+# Prior art
+[prior-art]: #prior-art
+
+[ExternalDNS](https://github.com/kubernetes-sigs/external-dns)
+
+* Uses annotations on the target Gateway as opposed to a proper API.
+* Requires access to the HTTP route resources.
+* Supports only a single provider per external dns instance.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+When a provider is configured using a kind not supported by MGC e.g. `ExternalDNS` we will be relying on an external controller to correctly update the status of any DNSRecord resources created by our policy. This may have a negative impact on our ability to correctly report status back to the target resource.
```

Comment:
```suggestion
When a provider is configured using a kind not supported by the `DNSPolicy` controller e.g. `ExternalDNS` we will be relying on an external controller to correctly update the status of any DNSRecord resources created by our policy. This may have a negative impact on our ability to correctly report status back to the target resource.
```